ironfish/src/primitives/rawTransaction.test.ts

```diff
@@ -84,6 +84,10 @@ describe('RawTransaction', () => {
     const mintedValue = valuesByAsset.get(asset.id())
     Assert.isNotUndefined(mintedValue)
     expect(mintedValue).toEqual(1n)
+
+    // should have same size for posted transaction and estimated size from raw transaction
+    const serializedPost = posted.serialize()
+    expect(raw.postedSize(account.publicAddress)).toEqual(Buffer.byteLength(serializedPost))
```

Comment:
Since `serializedPost` is already a Buffer, let's prefer calling byteLength directly:

```suggestion
    expect(raw.postedSize(account.publicAddress)).toEqual(serializedPost.byteLength)
```